polling_stations/templates/api_docs/blueprints/finder.apibp

```diff
@@ -159,7 +159,7 @@ the following conditions can be observed in the response body:
         { "detail": "Could not geocode from any source" }
 
 
-## Address search [/address/{slug}.json]
+## Address search [/address/{uprn}.json]
```

Comment:
:books: Nice :+1:

Worth noting that this will also have a knock-on effect for the devs.DC API docs as that will inherit this change. 

I think we should probably link to an explanation of what a UPRN (especially in the devs.DC API docs). In general, our API users probably won't have an AddressBase licence, but we have had conversations with at least one (potential) user (JISC) who are interested in querying by UPRN so its useful to explain you can do that, but also note that if you don't have it, its just.. a number basically.

Reply:
https://github.com/DemocracyClub/UK-Polling-Stations/issues/2918